exporters/otlp/src/test/java/io/opentelemetry/exporter/otlp/OtlpGrpcSpanExporterTest.java

```diff
@@ -34,9 +42,42 @@
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
-import org.mockito.Mockito;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
 class OtlpGrpcSpanExporterTest {
+
+  @RegisterExtension
+  public static ServerExtension server =
```

Comment:
This are great test, but why adding them in a PR that says it does make things final?

Reply:
Without the spy (can't because of final), we lose a significant test that the key-value splitting logic is correct (a regression test since it was broken for a while).